### ISSUE TITLE
[Unit 4][Topic Principles of accessible design] Homework Idea too early in unit

### ISSUE THREAD

**Issue by daniel-montalvo** [w3c/wai-curricula#91]
According to Lewis:

> The homework for this topic includes finding a web page with a failure for a standard, but Topic: First Checks for Web Accessibility where accessibility checks are covered is later in the unit. I think this one is too early in the unit.

**Comment by AndrewArch** [comment]
Had the same thought myself :)

**Comment by daniel-montalvo** [comment]
The whole homework and teaching ideas have been reworked. This issue has been adressed now. Feel free to reopen if necessary
